changelog/unreleased/change-remove-privacy-url-and-imprint-url-from-config

```diff
@@ -0,0 +1,7 @@
+Change: remove PrivacyURL and ImprintURL from the config
+
+We've removed PrivacyURL and ImprintURL from the config, hence these were living in Web config options,
```

Comment:
can you do a different phrasing? It sounds a bit confusing :)

Reply:
`We've removed the option PrivacyURL and ImprintURL from the config, since other clients weren't able to consume these`

Reply:
e.g.